aldryn_faq/models.py

```diff
@@ -46,10 +46,16 @@ def get_slug_in_language(record, language):
 
 @python_2_unicode_compatible
 @version_controlled_content(follow=['appconfig'])
-class Category(TranslationHelperMixin, TranslatableModel):
+class Category(TranslatedAutoSlugifyMixin, TranslationHelperMixin,
```

Comment:
Out of scope for this branch, but we should probably move to using python-slugify everywhere (translation-tools still uses django slugification iirc).